components/tidb_query_datatype/src/codec/data_type/chunked_vec_bytes.rs

```diff
@@ -44,8 +44,7 @@ impl ChunkedVecBytes {
     }
     #[inline]
     pub fn get(&self, idx: usize) -> Option<BytesRef> {
-        assert!(idx < self.len());
-        if self.bitmap.get(idx) {
+        if self.bitmap.get(idx).unwrap() {
```

Comment:
If we can make sure `idx` will not out of bounds, we can use `self.bitmap[idx]` instead.

Reply:
missed the `Index` implementation 🤦  I'll add the assert again

```suggestion
        assert!(idx < self.len());
        if self.bitmap.get[idx] {
```